spec/features/welcome_page_spec.rb

```diff
@@ -21,5 +21,14 @@
       expect(page).to have_link("Dashboard")
     end
 
+    it "Podsmack button takes you to home page" do 
+      visit "/"
+
+      click_on('Find your next favorite podcast')
+      expect(current_path).to eq('/podcasts')
+      click_on('Podsmack')
+      expect(current_path).to eq('/')
+    end
```

Comment:
Previously this redirected to top of a page, this now redirects to the home page.